src/systems/filecoin_mining/storage_mining/storage_miner_actor.go

```diff
@@ -2,7 +2,20 @@ package storage_mining
 
 import base_mining "github.com/filecoin-project/specs/systems/filecoin_mining"
 import sector "github.com/filecoin-project/specs/systems/filecoin_mining/sector"
+import sealing "github.com/filecoin-project/specs/systems/filecoin_mining/sector"
 
-func (sm *StorageMinerActor_I) SubmitPoSt(postProof base_mining.PoStProof, sectorStateSets sector.SectorStateSets) {
+func (sma *StorageMinerActor_I) SubmitPoSt(postProof base_mining.PoStProof, sectorStateSets sector.SectorStateSets) {
+	panic("TODO")
+}
+
+func (sma *StorageMinerActor_I) CommitSector(onChainInfo sealing.OnChainSealVerifyInfo) {
```

Comment:
shouldn't this list the deals in the sector?

Reply:
`OnChainSealVerifyInfo` contains a list of `DealID`